src/mavoscript.js

```diff
@@ -148,6 +148,7 @@ var _ = Mavo.Script = {
 		},
 		"divide": {
 			scalar: (a, b) => a / b,
+			leftUnary: b => b,
```

Comment:
I looked up our discussion on [PR #361](https://github.com/mavoweb/mavo/pull/361) and David said that the `leftScalar` should be invoked when the left argument/operand is missing. Although the left and right unaries are derived from their respective scalars, the name `leftUnary` does seem to have the implication that the unaryOp is done on the **left** argument and when the **right** argument is missing. Different people might have different interpretations though.

Reply:
After some thought, I'm pretty sure this is confusing and `leftUnary` should be for when the right argument is missing. Since @karger did not express any thoughts to the opposite, let's do it.